native/acter/src/api/attachments.rs

```diff
@@ -388,6 +387,10 @@ impl AttachmentsManager {
         self.room.room_id().to_string()
     }
 
+    pub fn object_id_str(&self) -> String {
```

Comment:
the managers for comments and attachments didn't give us a way to subscribe to the parent object, so I added these helpers from the rust side.